crux-core/src/crux/api/ICruxAPI.java

```diff
@@ -98,27 +98,15 @@
     public Map<Keyword,?> status();
     // TODO elaborate
 
-    /**
-     * Checks if a submitted tx did update an entity.
-     *
-     * @param submittedTx must be a map returned from {@link
-     * #submitTx(List txOps)}.
-     * @param eid         an object that can be coerced into an entity id.
-     * @return            true if the entity was updated in this transaction.
-     */
-    public boolean hasSubmittedTxUpdatedEntity(Map<Keyword,?> submittedTx, Object eid);
 
     /**
-     * Checks if a submitted tx did correct an entity as of valid
-     * time.
+     * Checks if a submitted tx was successfully committed.
      *
-     * @param submittedTx  must be a map returned from {@link
+     * @param submittedTx must be a map returned from {@link
      * #submitTx(List txOps)}.
-     * @param validTime    valid time of correction to check.
-     * @param eid          an object that can be coerced into an entity id.
-     * @return             true if the entity was updated in this transaction.
+     * @return            true if the submitted transaction was committed.
      */
-    public boolean hasSubmittedTxCorrectedEntity(Map<Keyword,?> submittedTx, Date validTime, Object eid);
+    public boolean hasTxCommitted(Map<Keyword,?> submittedTx);
```

Comment:
I'd specify that this method `throws NodeOutOfSyncException` (and add it as `@throws NodeOutOfSyncException <...>` to the Javadoc) - it's an unchecked exception, so the user doesn't _have_ to handle it, but good to put it there as documentation